scala_modules/events/src/main/scala/io/dagster/events/EventPipeline.scala

```diff
@@ -98,24 +92,50 @@ object EventPipeline {
     )
 
     // Create an ADT StorageBackend to abstract away which we're talking to
-    val backend: StorageBackend = (conf.localPath, conf.s3Bucket, conf.s3Prefix) match {
-      case (None, Some(bucket), Some(prefix)) => S3StorageBackend(bucket, prefix)
-      case (Some(path), None, None) => LocalStorageBackend(path)
+    val backend: StorageBackend = (conf.localPath, conf.s3Bucket, conf.s3Prefix, conf.date) match {
+      case (None, Some(bucket), Some(prefix), date) => S3StorageBackend(bucket, prefix, date)
+      case (Some(path), None, None, date)           => LocalStorageBackend(path, date)
       case _ => throw new IllegalArgumentException("Error, invalid arguments")
     }
 
     val events = readEvents(backend, conf.date)
 
-    // Print a few records
+    // Print a few records in debug logging
     events
       .take(20)
-      .foreach(println)
+      .foreach(log.debug)
+
+    // Ensure output path is empty
+    backend match {
+      case l: LocalStorageBackend => {
```

Comment:
is there a scala convention indicating a variable is unused as, ``l`` is?

Reply:
I also generally avoid lowercase l as a variable name as it looks like 1.

`val l = 1`

etc

Reply:
yep, thx for catching this one too - idiomatic is to use _ for unused

agree on use of lowercase `l`, although for pattern matching, the convention in Scala is to use the lowercase of the first character of the type. Will be obviated in this case since I'm not using `l`...